lib/Parser/TypesParser.php

```
@@ -14,15 +14,14 @@ public function parseTypes(string $types): DocblockTypes
         }
 
         $types = str_replace('&', '|', $types);
-        $types = explode('|', $types);
         $docblockTypes = [];
 
-        foreach ($types as $type) {
-            $type = trim($type);
+        foreach (explode('|', $types) as $type) {
+            $type = trim($type, "? \t\n\r\0\x0B");
```

[`trim()`'s default mask](https://www.php.net/trim) is `"" \t\n\r\0\x0B""`.